src/main/ui/src/App.js

```diff
@@ -1,26 +1,28 @@
-import ParkNotes from './components/Notes/ParkNotes';
-import {useState} from 'react';
+import ParkNotes from "./components/Notes/ParkNotes";
+import { useState } from "react";
+import "./App.css";
+import { Header } from "./components/header/Header";
 
-    const App = () => {
-        const[notes, setNotes] = useState([
-        ]);
+const App = () => {
```

Comment:
When placed here the `Header` component will not render. Please move into the spot where it will.